app/controllers/users_controller.rb

```diff
@@ -1,5 +1,9 @@
 class UsersController < ApplicationController
-  def index; end
+  def index
+    @user = User.all
```

Comment:
- [ ] [OPTIONAL] Since you are getting all the users I think it would be better to change your variable to `@users` to make more sense and make your code understandable